torch_trainer.py

```diff
@@ -177,6 +176,9 @@ def train(self):
 
         if 'val' not in self.datasets:
             logging.info('No validation dataset is provided. Train without vaildation.')
+            from pytorch_lightning.callbacks.early_stopping import EarlyStopping
+            self.trainer.callbacks = \
+                [callback for callback in self.trainer.callbacks if not isinstance(callback, EarlyStopping)]
```

Comment:
Why apply early stopping when no validation is provided?

Reply:
This logic is to keep the callbacks that are **not** early stopping if no validation provided.